packages/i18n/src/messages/localizedConstants.tsx

```diff
@@ -1929,3 +1953,65 @@ export const getEvaluatedLanguageAbility = (
     ability,
     `Invalid evaluatedLanguageAbility ${ability}`,
   );
+
+const assessmentStepTypes = defineMessages({
```

Comment:
Shoot, I requested the translation for these, too.  Should be interesting to see how close they come.

Reply:
Got 'em in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8169/commits/eba8d4a61ee1bbd8f0974eb253d3550f9b8eccad
Take a look if you'd like